Lesson26-part2/index.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+
+// Define a evenNumbers variable and assign a function as value
+// When executed the evenNumbers function will print even numbers between 0 and 100 as output
+// Call this function 5 times
+let evenNumbers = function() {
+  for (let number = 0; number <= 100; number++) {
+    number % 2 == 0 ? console.log(number) : '';
```

Comment:
Do not write code like this @marcelomatosdev I know it works but you have compromised readability. A simple 
`if (number % 2 == 0) { console.log(number); }` is more readable and not confusing

Reply:
You're right! Thanks!